packages/node/index.js

```diff
@@ -1,6 +1,10 @@
-const request = require('r2');
+const fetch = require('node-fetch');
```

Comment:
`r2` seems to be a mothballed project at this point so I'm moving us over to using `fetch()` instead.